django-stubs/http/cookie.pyi

```diff
@@ -1,5 +1,8 @@
+from http import cookies
```

Comment:
```suggestion
from http.cookies import SimpleCookie as SimpleCookie
```

Reply:
That does not fix the issue though since then it will still we generic.